pkg/network/kube_proxy.go

```diff
@@ -206,8 +209,16 @@ func renderStandaloneKubeProxy(conf *operv1.NetworkSpec, bootstrapResult *bootst
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["KubeProxyImage"] = os.Getenv("KUBE_PROXY_IMAGE")
 	data.Data["KubeRBACProxyImage"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
-	data.Data["KUBERNETES_SERVICE_HOST"] = bootstrapResult.Infra.APIServers[bootstrap.APIServerDefault].Host
-	data.Data["KUBERNETES_SERVICE_PORT"] = bootstrapResult.Infra.APIServers[bootstrap.APIServerDefault].Port
+	hsc := hypershift.NewHyperShiftConfig()
+	//TODO (relyt0925): when hypershift appropriately signs kube-apiserver certificate with node local
+	//address it is my recommendation to move to using the node local loadbalancer for all workload
+	if hsc.Enabled && bootstrapResult.Infra.PlatformType == v1.IBMCloudPlatformType {
```

Comment:
I assumed that this should be applicable to all hypershift deployments, not only `IBMCloudPlatformType`?